### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by trevorbox**

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -8,4 +8,6 @@ on:
 jobs:
   shared-operator-workflow:
     name: shared-operator-workflow
-    uses: redhat-cop/github-workflows-operators/.github/workflows/pr-operator.yml@v1.0.0
+    uses: trevorbox/github-workflows-operators/.github/workflows/pr-operator.yml@main
```
Comment (trevorbox) [inline .github/workflows/pr.yaml:11]: @raffaelespazzoli See https://github.com/redhat-cop/github-workflows-operators/pull/7 
I think there is an opportunity to update the shared workflow to add testing.

File: Makefile [hunk 2]
```diff
@@ -220,15 +220,102 @@ helmchart: kustomize
...
+
+.PHONY: helm
+HELM = ./bin/helm
+helm: ## Download helm locally if necessary.
+ifeq (,$(wildcard $(HELM)))
+ifeq (,$(shell which helm 2>/dev/null))
+	echo "Downloading ${HELM}."
+	OS=$(shell go env GOOS) ;\
+	ARCH=$(shell go env GOARCH) ;\
+	curl --create-dirs -sSLo ${HELM}.tar.gz https://get.helm.sh/helm-v3.8.0-$${OS}-$${ARCH}.tar.gz ;\
```
Comment (trevorbox) [inline Makefile:286]: Parameterize the Helm version

**Comment by raffaelespazzoli** [comment]
sidecar with vault initialization:    https://github.com/raffaelespazzoli/vault-config-operator/blob/artifactory/config/local-development/vault-values.yaml

**Comment by trevorbox** [comment]
> sidecar with vault initialization: https://github.com/raffaelespazzoli/vault-config-operator/blob/artifactory/config/local-development/vault-values.yaml

This is implemented.

**Pull request merged**